File: python/email/src/email_sender.py
from logger import logger

def send_emails(boletos):
    for boleto in boletos:
        logger.info(f"Enviando email para boleto ID: {boleto['id']}, Nome: {boleto['name']}")
        # Aqui iriam as chamadas de envio de email reais
    return [boleto['id'] for boleto in boletos]

File: go_teste_estudo/src/email_sender.go
package main

import "log"

func SendEmails(boletos []Boleto) []int {
    var boletoIDs []int
    for _, boleto := range boletos {
        log.Printf("Enviando email para boleto ID: %d, Nome: %s", boleto.ID, boleto.Name)

        boletoIDs = append(boletoIDs, boleto.ID)
    }
    return boletoIDs
}